client/utilities/rfc-index.txt.test.ts

```diff
@@ -0,0 +1,321 @@
+// @vitest-environment nuxt
+import fs from 'node:fs'
+import path from 'node:path'
+import { vi, describe, beforeEach, afterEach, test, expect } from 'vitest'
+import { renderRfcIndexDotTxt } from './rfc-index'
+import { ApiClient, SlugEnum } from '~/generated/red-client'
+
+const expectedContent = fs
+  .readFileSync(path.join(import.meta.dirname, 'rfc-index.txt'), 'utf-8')
+  .toString()
```

Comment:
tests rendering against a wget of the existing rfc-index.txt truncated to RFC0013